spec/controllers/attributes_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require Rails.root.join('spec', 'support', 'mock_auth_helper.rb')
+
+describe AttributesController do
```

Comment:
I still disagree.  Controllers are part of the application and have methods that are called, so testing them is important.  Testing them helps verify they are thin and don't have too much logic.  Integration tests aren't what I want because I simply want to test my controllers call the right methods and pass the right values.